internal/sherlock/sherlock_integration_test.go

```diff
@@ -446,7 +446,9 @@ func integrationSetup(t *testing.T) {
 	// when running tests workdir is the package directory ie cmd/server
 	// so a relative path to changelogs is needed.
 	// TODO cleaner method to supply path to changelogs and run migration in tests
-	if err := db.ApplyMigrations("../../db/migrations", sherlock.Config); err != nil {
+	if err := db.ApplyMigrations("../../db/migrations", sherlock.Config); err == migrate.ErrNoChange {
```

Comment:
This comment from the linter is what is causing all your compilation failures. Seems like a missing import statement.. I'd like to see the CI checks passing before doing a full review.

Reply:
yep, thanks! Sorry I got my test to pass in vscode, forgotting to run the full suite and then just went punted debugging till today.